.github/workflows/flutter-build.yml

```diff
@@ -41,3 +42,4 @@ jobs:
         working-directory: ./client/flutter
       - run: flutter build appbundle
         working-directory: ./client/flutter
+      - run: git diff --exit-code
```

Comment:
do we need this for Android if this is only meant to catch Podfile updates? probably not a problem to have it here / not a blocker, just checking

Reply:
This catches updates both to `Podfile` and also for `pubspec.lock` (which will matter for Android builds). That being said, I'm not sure if there are any Android-specific files that flutter updates.